src/components/List/List.jsx

```diff
@@ -1,22 +1,59 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import db from '../lib/firebase';
-import { collection, doc, updateDoc } from 'firebase/firestore';
+import db from '../../lib/firebase';
+import { collection, doc, updateDoc, query, orderBy } from 'firebase/firestore';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import moment from 'moment';
-
-import { getEstimate, calcTimeDiff, formatDate } from '../helpers';
+import './list.css';
+import {
+  getEstimate,
+  calcTimeDiff,
+  formatDate,
+  calcDaysSince,
+} from '../../helpers';
 
 export const List = ({ token }) => {
   let navigate = useNavigate();
-  const [value, loading, error] = useCollection(collection(db, token), {
+  const q = query(collection(db, token), orderBy('item'));
+  const [value, loading, error] = useCollection(q, {
     snapshotListenOptions: { includeMetadataChanges: true },
   });
+
   let [filterText, setFilterText] = useState('');
+  let [items, setItems] = useState([]);
+
+  useEffect(() => {
+    if (value) {
+      let arr = [];
+      value.docs.forEach((item) => {
+        arr.push({ ...item.data(), id: item.id });
+      });
+      arr.map((item) => (item.isActive = handleActive(item)));
+      arr.sort(
+        (itemA, itemB) =>
+          itemA.estimated_next_purchase - itemB.estimated_next_purchase,
+      );
+      arr.sort(
+        (itemA, itemB) => Number(itemB.isActive) - Number(itemA.isActive),
+      );
+      setItems(arr);
+    }
+  }, [value]);
+
+  const handleActive = (item) => {
+    if (
+      calcDaysSince(item.last_purchased_date) >
+      item.estimated_next_purchase * 2
+    ) {
+      return false;
+    }
+    return true;
+  };
 
   const updateDocument = async (document) => {
     const docRef = doc(db, token, document.id);
-    let docData = document.data();
+    let docData = document;
+    console.log(docData);
```

Comment:
Do we need to keep logging this? It's generally a good idea to remove logging that was added for debugging. If it's a log that would be useful to have long term, you might consider adding some static text to give some context around the data that's being logged.